src/System.Data.SqlClient/src/Interop/SNINativeMethodWrapper.Windows.cs

```diff
@@ -182,9 +182,15 @@ internal struct SNI_Error
 
         #region DLL Imports
         [DllImport(SNI, CallingConvention = CallingConvention.Cdecl, EntryPoint = "SNIAddProviderWrapper")]
+#if NET8_0_OR_GREATER
```

Comment:
Why are these getting obsoleted if they're marked as `internal` or `private` APIs? Mainly asking since the APIs at the beginning of this file are also `internal` or `private` but they were not decorated with the attribute.

Reply:
Its done mainly to resolve warning CS0618, I can suppress warning alternatively.